physionet-django/static/sample/create-course-schema.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
We have these samples, but we actually don't seem to use them? Only the hard-coded JSON field is used in the guidelines page. I do this it's nicer to have these as static assets in this folder which we then call an include to show on the guidelines page.